src/CommunityToolkit.Maui.Core/Essentials/FileSaver/FileSaver.shared.cs

```diff
@@ -11,6 +13,11 @@ public static class FileSaver
 	public static IFileSaver Default => defaultImplementation.Value;
 
 	/// <inheritdoc cref="IFileSaver.SaveAsync(string, string, Stream, CancellationToken)"/>
+	[SupportedOSPlatform("Android26.0")]
```

Comment:
Attributes are not needed, as it works on Android 26 and lower. You added info for devs that initialPath doesn’t work on Android 26 and lower

Reply:
I would keep the attributes to have a warning if a dev uses the method with target lower than api 26 on android, if that is ok. So it's likely that the dev uses the methods without initialUri. Is that ok?

Reply:
Don’t worry I will finish the PR by the end of the week

Reply:
Ha - I actually agree with both of you. 

Let's keep the attributes to note that the feature "isn't supported" on Android API 25 and below. 

I can see someone opening an Issue reporting a "bug" in the future after finding out this different behavior on Android 25, for example. And with this attribute here, we can easily close the Issue noting that it is only "supported" on Android 26+